wampy/message_handler.py

```diff
@@ -75,10 +75,15 @@ def handle_challenge(self, message_obj):
             return
 
         secret = os.environ['WAMPYSECRET']
-        challenge_data = message_obj.challenge
-        signature = compute_wcs(secret, str(challenge_data))
+        if message_obj.auth_method == 'ticket':
+            logger.debug("proceeding with ticket authentication method")
+            message = Authenticate(secret)
+        else:
```

Comment:
I'm happy enough to merge this as it is, but now you've made me aware of the different auth methods `wampy` should support, I'd eventually prefer to see an `elseif` here conditioning against `wampcra`, then the hanging `else` raising a `RuntimeError` or something, as at some point someone else will play about with another auth method. I'll give you some time to think.

Also, going forward I'd like to use `Enum` to martial these strings into specific types. So something like...
```
class AuthMethod(Enum):
    ticket = 'ticket'
    cra = 'wampcra'
```
would look great in `wampy.constants.py`!

Reply:
My point here was to add explicit support for the `ticket` auth method while maintaining backwards compatibility as much as possible.

I agree with the Enum and `RuntimeError` approach, but I'm not sure when I could get into it. Possibly within a month, but you know how it goes 😀